protocol-specs/generators/dpos_delegate_shuffling/index.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const { hash } = require('@liskhq/lisk-cryptography');
+const BaseGenerator = require('../base_generator');
+const previousDelegateList = require('./delegate_address_list.json').list;
+
+const generateShuffledDelegateList = () => {
+	const previousRoundSeed1 = 'b9acc2f1fda3666bfb34107f1c6dccc4';
+	const delegateList = [...previousDelegateList];
+	for (const delegate of delegateList) {
+		const addressBuffer = Buffer.alloc(8);
+		addressBuffer.writeBigUInt64BE(BigInt(delegate.address.slice(0, -1)));
+		const seedSource = Buffer.concat([
+			Buffer.from(previousRoundSeed1, 'hex'),
+			addressBuffer,
+		]);
+		/* eslint-disable dot-notation */
+		delegate['roundHash'] = hash(seedSource);
```

Comment:
btw why not `dot` notation here?

Reply:
I changed it, I had TS in my mind before